### ISSUE TITLE
Scipy emits warning on Windows

### ISSUE THREAD

**Issue by Hoxbro** [conda-forge/scipy-feedstock#264]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

With the latest release of Scipy our CI is starting to emit a warning the first time it is imported on Windows. I can recreate this in a VM. I haven't checked if everything works as expected, or there is a bigger problem. 

CI:
https://github.com/holoviz/holoviews/actions/runs/7608997256/job/20719330811#step:4:974

In VM:
![ Screenshot 2024-01-22 10 59 12](https://github.com/conda-forge/scipy-feedstock/assets/19758978/cd1ec72e-ea6d-4d23-bf90-14cb3c9f022d)

### Installed packages

```shell
bzip2                     1.0.8                hcfcfb64_5    conda-forge
ca-certificates           2023.11.17           h56e8100_0    conda-forge
intel-openmp              2024.0.0         h57928b3_49840    conda-forge
libblas                   3.9.0              18_win64_mkl    conda-forge
libcblas                  3.9.0              18_win64_mkl    conda-forge
libexpat                  2.5.0                h63175ca_1    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libhwloc                  2.9.3           default_haede6df_1009    conda-forge
libiconv                  1.17                 hcfcfb64_2    conda-forge
liblapack                 3.9.0              18_win64_mkl    conda-forge
libsqlite                 3.44.2               hcfcfb64_0    conda-forge
libxml2                   2.12.4               hc3477c8_1    conda-forge
libzlib                   1.2.13               hcfcfb64_5    conda-forge
mkl                       2022.1.0           h6a75c08_874    conda-forge
numpy                     1.26.3          py312h8753938_0    conda-forge
openssl                   3.2.0                hcfcfb64_1    conda-forge
pip                       23.3.2             pyhd8ed1ab_0    conda-forge
pthreads-win32            2.9.1                hfa6e2cd_3    conda-forge
python                    3.12.1          h2628c8c_1_cpython    conda-forge
python_abi                3.12                    4_cp312    conda-forge
scipy                     1.12.0          py312h8753938_0    conda-forge
setuptools                69.0.3             pyhd8ed1ab_0    conda-forge
tbb                       2021.11.0            h91493d7_0    conda-forge
tk                        8.6.13               h5226925_1    conda-forge
tzdata                    2023d                h0c530f3_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
vc                        14.3                hcf57466_18    conda-forge
vc14_runtime              14.38.33130         h82b7239_18    conda-forge
vs2015_runtime            14.38.33130         hcb4865c_18    conda-forge
wheel                     0.42.0             pyhd8ed1ab_0    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
```

### Environment info

```shell
active environment : tmp3
    active env location : C:\Users\shh\miniconda3\envs\tmp3
            shell level : 3
       user config file : C:\Users\shh\.condarc
 populated config files : C:\Users\shh\miniconda3\.condarc
          conda version : 23.7.2
    conda-build version : not installed
         python version : 3.10.12.final.0
       virtual packages : __archspec=1=x86_64
                          __win=0=0
       base environment : C:\Users\shh\miniconda3  (writable)
      conda av data dir : C:\Users\shh\miniconda3\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : C:\Users\shh\miniconda3\pkgs
                          C:\Users\shh\.conda\pkgs
                          C:\Users\shh\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\shh\miniconda3\envs
                          C:\Users\shh\.conda\envs
                          C:\Users\shh\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/23.7.2 requests/2.31.0 CPython/3.10.12 Windows/10 Windows/10.0.22621
          administrator : False
             netrc file : None
           offline mode : False
```

**Comment by rgommers** [comment]
https://github.com/scipy/scipy/pull/19945 should fix this.

**Comment by h-vetinari** [comment]
Thanks for the report @Hoxbro, and for the quick fix @rgommers! 🙏 

IMO a warning on every import is definitely enough noise to merit a fix, so I've done that in #265
